spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/rxjava/SleuthRxJavaSchedulersHook.java

```diff
@@ -77,16 +84,31 @@ public Action0 onSchedule(Action0 action) {
 		private Tracer tracer;
 		private TraceKeys traceKeys;
 		private Span parent;
+		private final List<String> threadsToIgnore;
 
-		public TraceAction(Tracer tracer, TraceKeys traceKeys, Action0 actual) {
+		public TraceAction(Tracer tracer, TraceKeys traceKeys, Action0 actual,
+				List<String> threadsToIgnore) {
 			this.tracer = tracer;
 			this.traceKeys = traceKeys;
+			this.threadsToIgnore = threadsToIgnore;
 			this.parent = tracer.getCurrentSpan();
 			this.actual = actual;
 		}
 
+		@SuppressWarnings("Duplicates")
 		@Override
 		public void call() {
+			// don't create a span if the thread name is on a list of threads to ignore
+			for (String threadToIgnore : this.threadsToIgnore) {
+				String threadName = Thread.currentThread().getName();
+				if (threadName.matches(threadToIgnore)) {
+					log.debug(String.format(
```

Comment:
argh.. sorry didn't realize this is apache commons logging (which doesn't have a deferred format string api).